components/brave_new_tab_ui/storage/new_tab_storage.ts

```diff
@@ -50,7 +50,13 @@ export const defaultState: NewTab.State = {
   },
   togetherPromptDismissed: false,
   rewardsState: {
-    adsEstimatedEarnings: 0,
+    adsStatement: {
```

Comment:
I would just try out a quick upgrade case, since the old `adsEstimatedEarnings` might be sitting around in  `localStorage` and we're not going to read it anymore.

Reply:
Tested upgrade path and working as expected